curve-ansible/roles/generate_config/templates/s3.conf.j2

```diff
@@ -10,7 +10,7 @@ s3.sk={{ s3_sk }}
 # http = 0, https = 1
 s3.http_scheme={{ s3_http_scheme }}
 s3.verify_SSL={{ s3_verify_ssl }}
-s3.user_agent_conf={{ s3_user_agent_conf }}
```

Comment:
Ansible is no longer maintained and can not be modified

Reply:
Ok.